plugin/dashboard.lua

```diff
@@ -40,9 +40,14 @@ if db.session_auto_save_on_exit then
   api.nvim_create_autocmd('VimLeavePre', {
     group = session_auto_save,
     callback = function()
-      pcall(vim.cmd, "NvimTreeClose")
       if db_session.session_exists() and vim.fn.len(vim.fn.getbufinfo({ buflisted = 1 })) > 1 then
```

Comment:
no I mean 
```lua
vim.api.nvim_exec_autocmd('User', 
      pattern = 'SessionSavePre',
      modeline= false,
)
 
```

Reply:
Ah okay :D will do :+1:

Reply:
This should do it. I also added documentation.